src/main/java/com/opentable/db/postgres/embedded/PreparedDbProvider.java

```diff
@@ -196,25 +215,46 @@ private static void create(final DataSource connectDb, final String dbName, fina
         }
     }
 
-    private static class DbInfo
+    public static class DbInfo
```

Comment:
If this is going to be part of the public API for this library, what do you think of un-nesting it into a top-level class?

Reply:
I'm agnostic about that and will defer to @stevenschlansker  I kind of deliberately left it ugly.